conversation_classification/kaggle/model.py

```diff
@@ -209,94 +274,83 @@ def main():
       config=tf.contrib.learn.RunConfig(
         tf_random_seed=TRAIN_SEED,
       ),
-      model_dir=None)
+      model_dir=FLAGS.model_dir)
 
     # Train model
     train_input_fn = tf.estimator.inputs.numpy_input_fn(
-      x={WORDS_FEATURE: x_train},
-      y=y_train,
+      x={WORDS_FEATURE: data.x_train},
+      y=data.y_train,
       batch_size=BATCH_SIZE,
       num_epochs=None, # Note: For training, set this to None, so the input_fn
                        # keeps returning data until the required number of train
                        # steps is reached.
       shuffle=True)
-
-    classifier.train(input_fn=train_input_fn, steps=TRAIN_STEPS)
+    classifier.train(input_fn=train_input_fn, steps=FLAGS.train_steps)
 
     # Predict on held-out test data
     test_input_fn = tf.estimator.inputs.numpy_input_fn(
-      x={WORDS_FEATURE: x_test},
-      y=y_test,
+      x={WORDS_FEATURE: data.x_test},
+      y=data.y_test,
       num_epochs=1,     # Note: For evaluation and prediction set this to 1,
                         # so the input_fn will iterate over the data once and
                         # then raise OutOfRangeError
       shuffle=False)
-
     predicted_test = classifier.predict(input_fn=test_input_fn)
     test_out = pd.DataFrame(
       [(p['classes'], p['probs'][1]) for p in predicted_test],
       columns=['y_predicted', 'prob']
     )
-    test_out['comment_text'] = x_train_text
-    test_out['y_true'] = y_test
-
-    # Write out predictions and probabilities for test data
-    tf.logging.info("Writing test predictions to {}".format(TEST_OUT_PATH))
-    test_out.to_csv(TEST_OUT_PATH)
+    test_out['comment_text'] = data.x_train_text
```

Comment:
I think this should be data.x_test_text

Reply:
Yes, great catch. I don't actually use that field anywhere so I'll delete those lines. I used make predictions on the test data and write it to file, but I've remove that.